front-end/src/components/Chart.js

```diff
@@ -31,7 +35,7 @@ const Chart = ({ chartInformation }) => {
             enabled: false
         },
         boost: {
-            enabled: true
+            enabled: false
```

Comment:
We should add checks to disable this when using video... why did we do this again?

Reply:
Added video as a chartInformation type and used that to dynamically set boosting